register.php

```diff
@@ -19,15 +19,52 @@
 
 include_once("sqlconnect.php");
 include_once("helpers.php");
+include_once("mail_config.php");
+
 if(!$_POST['username'] || !$_POST['password'] || !$_POST['privatekey'] || !$_POST['publickey'])
   die("Error - one of the parameters is not set.");
 
 //check if user already exists
 if(userExists($conn, $_POST['username']))
   die("Error - user already exists. Try to login instead.");
 
+if ($use_mail_registration) {
```

Comment:
It's really just nitpicking, but the rest of the code doesn't seem to have a space after if, so keeping that convention would be ideal.